homeassistant/components/mastodon/quality_scale.yaml

```diff
@@ -0,0 +1,102 @@
+rules:
+  # Bronze
+  action-setup:
+    status: done
+    comment: |
+      Legacy Notify platform can't be created within async-setup.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow:
+    status: done
+    comment: |
+      YAML flow deprecated and will be removed in 2025.2.0
+  dependency-transparency: done
```

Comment:
Mastodon.py isn't built and published in a CI

Reply:
It does have reference to CircleCI passing (bottom of repo page https://github.com/halcy/Mastodon.py) so not as transparent as GitHub actions but seems to be there.  Is this enough to pass?

Not making bronze because of dependency CI is tough (general observation rather than just this one).

Reply:
It seems there are only tests run on CircleCI, not the build and publishing of the package.

Reply:
Thanks for the clarification, Mastodon.py is the most well supported lib out there and I don't want to build/maintain my own.

I know this is just one case but be interesting to see if this rule stops many integrations meeting bronze criteria.